### DISCUSSION TITLE
Welcome to Rentalx Discussions!

### DISCUSSION THREAD

**Discussion by ermessonlima** [ermessonlima/Rentalx#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ermessonlima** [comment, marked as answer]
Hello my friends!

**Reply by LelioH** [reply]
Hello! How's it going?

**Comment by oliveiragabriel1** [comment]
Hello, how I can help?

**Reply by ermessonlima** [reply]
Send pix!

**Reply by oliveiragabriel1** [reply]
Dont have money my friend

**Comment by LelioH** [comment]
Hi there guys!

**Reply by ermessonlima** [reply]
Hello!

**Reply by oliveiragabriel1** [reply]
Hello

**Comment by oliveiragabriel1** [comment]
Six so good?

**Reply by LelioH** [reply]
Great
